README.md

```diff
@@ -33,3 +33,43 @@ yarn dev
 yarn build
 ```
 
+## E2E Testing
+
+For E2E testing, we utilize [chopsticks](https://github.com/AcalaNetwork/chopsticks) and [playwright](https://playwright.dev/) to mandatorily write test cases that test all of business logic.
+
+### Setup chopsticks
+```bash
+npx @acala-network/chopsticks@latest xcm -p astar -p shiden
+```
+
+### Running tests locally in debug mode
+- Start the portal (the portal should run on http://localhost:8080)
+
+- You need to familiar with Playwright generator by running codegen
+```bash
+yarn playwright:codegen
+```
+
+- Run the following
+```bash
+yarn playwright
+```
+
+- debug each file individually
+```bash
+BASE_URL='http://localhost:8080' ENDPOINT='ws://127.0.0.1:8000' npx playwright test tests/dappstaking-transactions.spec.ts --project=chromium --debug
```

Comment:
How about change here to `yarn playwright tests/dappstaking-transactions.spec.ts` ?